TKeazirian.HTTPServer/Router/RoutesConfig.cs

```diff
@@ -51,6 +51,15 @@ public static Routes ConstructRoutes()
         routes.AddRoute("/kisses.gif",
             new Route(new List<HttpMethod> { HttpMethod.GET }, new ImageHandler())
         );
+        routes.AddRoute("/json-sample.json",
+            new Route(new List<HttpMethod> { HttpMethod.GET }, new FileServerHandler())
+        );
+        routes.AddRoute("/html-sample.html",
+            new Route(new List<HttpMethod> { HttpMethod.GET }, new FileServerHandler())
+        );
+        routes.AddRoute("/xml-sample.xml",
+            new Route(new List<HttpMethod> { HttpMethod.GET }, new FileServerHandler())
+        );
```

Comment:
Do you think this routing could be done dynamically? So instead of needing to add a specific route for any file, you could just have one route, like `resources/<file>` that would allow you to add and serve any file without needing to specify a new route for it?

Reply:
If you think this is do-able, I def think it could be a separate ticket

Reply:
I can certainly try 😄